source/code-snippets/usage-examples/updateMany.js

```diff
@@ -0,0 +1,34 @@
+// ignored first line
+const { MongoClient } = require("mongodb");
+
+// Replace the following with your MongoDB deployment's connection string.
+const uri =
+  "mongodb+srv://<user>:<password>@<cluster-url>?retryWrites=true&w=majority";
+
+const client = new MongoClient(uri);
+
+async function run() {
+  try {
+    await client.connect();
+
+    const database = client.db("sample_mflix");
+    const collection = database.collection("movies");
+
+    // create a filter to update all movies with a 'G' rating
+    const filter = { rated: "G" };
+    const options = {};
+    options.upsert = true; // create a document if no documents have a 'G' rating
```

Comment:
I think my issue with an upsert here is that creating a new document that looks like this:
```
{
   _id: 2,
   rated: 'G',
   num_mflix_comments: 2
}
```
is a slightly strange way for this method to behave on upsert. In reality you wouldn't ever want to create a movie with no title, no summary, etc.

It's not the end of the world, but I'd suggest changing it to a situation where you match a movie by title (where multiple movies share the same title, e.g. "Black Sheep") and update the rating or something. Just because in that case you're at least creating a movie with a title, so it's a little bit more of a valid document than one that just has a rating/num_comments after the upsert.

Reply:
Fixed - removed all instances of upsert